### ISSUE TITLE
Add ability to view all rows, not just up to 50

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#303]
```
What is the expected output? What do you see instead?

It would be helpful to be able to view all rows in a spreadsheet, and not just 
50 at a time. It makes it very difficult to clean all data at once if you have, 
say, 300 rows of data.

Right now, I just see options of 5 10 25 50. How about adding "All"?

What version of the product are you using? On what operating system?

2.0, Windows XP, Service Pack 3, Firefox

Please provide any additional information below.

```

Original issue reported on code.google.com by `Knight...@gmail.com` on 7 Jan 2011 at 3:53

**Comment by GoogleCodeExporter** [comment]
```
Working with more than your brain could absorb at a given time is the whole 
idea behind simplifying views in Refine by using Facets and Clustering.  Any 
given screen resolution can only display so many rows as well, and showing 300 
rows on the screen at one time doesn't resolve useful patterns the way you 
would like, trust us.  Please try to view the example [Screencasts] on the 
[GettingStarted] wiki page for examples of how to work with Facets and 
Clustering in general.
```

Original comment by `thadguidry` on 7 Jan 2011 at 5:07

**Comment by GoogleCodeExporter** [comment]
```
Thank you for your reply.

The thing is, we have data sets in which we actually need to be able to see all 
of them to meaningfully clean them. I promise. Our brains can absorb it.

By limiting it to 50, this only increases the number of steps it will take to 
clean our data. If we could have the option to see all of the rows of data, no 
matter how many (even in the 1000s), it would really help us be able to clean 
it.

A lot of it is trying to clean up text, like how countries have been spelled, 
or whether organizations have been abbreviated or spelled out. This isn't an 
issue of trying to clean numbers, really. 

So, the degree of difficulty is actually harder by this row limitation, and 
would be much easier if we could see all rows without limit.
```

Original comment by `Knight...@gmail.com` on 7 Jan 2011 at 8:31

**Comment by GoogleCodeExporter** [comment]
```
As a quick hack to get you unblocked, please dig into the source code directory 
and open this file: 
webapp/modules/core/scripts/view/data-table/data-table-view.js. Search for "50" 
in it, and add something like 1000. Be sure to have commas between the numbers. 
Then just shift-reload your browser showing Refine.
```

Original comment by `dfhu...@gmail.com` on 7 Jan 2011 at 8:40

**Comment by GoogleCodeExporter** [comment]
```
You REALLY need to watch the screencasts and see how the function 
http://code.google.com/p/google-refine/wiki/Clustering : CLUSTERING works.  You 
will be absolutely delighted in it's capability for your exact needs expressed, 
if you spend some time trying it out.
```

Original comment by `thadguidry` on 7 Jan 2011 at 8:42

**Comment by GoogleCodeExporter** [comment]
```
@dfhuynh - That's a great fix. Thanks!

@thadguidry - I will definitely get to know clustering better. Thanks to you, 
too!
```

Original comment by `Knight...@gmail.com` on 7 Jan 2011 at 10:39

**Comment by GoogleCodeExporter** [comment]
```
Obviously one can't see an arbitrarily large number of rows on a finite size 
screen, so it sounds like what you are actually requesting is navigation via 
the scroll bar vs paging.  Is that correct?  Are you still requesting this 
enhancement or have your needs been satisfied with Refine's currently available 
facilities.

```

Original comment by `tfmorris` on 14 Jun 2011 at 6:39
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
No response from requester.  Assuming their needs are satisfied.
```

Original comment by `tfmorris` on 18 Sep 2012 at 7:10
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
I would find it very handy to be able to navigate via the scroll bar. I often 
have projects with thousands of rows, and now it's quite time consuming to go, 
e.g., to line 3500 (by pushing 70 times on the 'next'-button). Or I am missing 
something here?
```

Original comment by `declercq...@gmail.com` on 8 Sep 2014 at 5:53
